### ISSUE TITLE
File not found: fresh revolve2 setup

### ISSUE THREAD

**Issue by andricmitrovic** [ci-group/revolve2#324]
Running simulation or brain optimization on Windows11/PyCharm/fresh anaconda env prints out the following in the console, but the code continues working:
> FatalError('File not found')
> Setting 'delete' parameter to False so that the xml can be saved

**Comment by surgura** [comment]
This is intended behaviour. On Windows the MuJoCo simulator has trouble with creating the temporary xml files which contains the simulation models. As a workaround we create it as a non-temporary file and then manually delete it after use. Downside is that if something crashes in between you are left with an unused file on your system. But this part of the code doesn't crash and the user stopping the program exactly on the right millisecond is very unlikely, so:

tldr
it's fine, don't worry

**Comment by surgura** [comment]
Task: improve the error message to say that these errors can be safely ignored.

**Comment by surgura** [comment]
Error message update merge to dev.
